SSA/Projects/InstCombine/LLVM/PrettyEDSLOverflow.lean

```diff
@@ -36,4 +37,16 @@ macro_rules
       | _ => Macro.throwUnsupported
     `([mlir_op| $[$resName =]? $opName ($xs,*) $[<{overflowFlags = #llvm.overflow< $ovflags,* >}>]? : ($argTys,*) -> ($retTy:mlir_type,*) ])
 
+syntax mlir_op_operand " = " MLIR.Pretty.overflow_int_cast_op mlir_op_operand (" overflow<" overflow_flag,* "> ")? " : " mlir_type " to " mlir_type : mlir_op
+macro_rules
+  | `(mlir_op| $resName:mlir_op_operand = $name:MLIR.Pretty.overflow_int_cast_op $x $[overflow< $[$f],* >]? : $t to $t' ) => do
+    let some opName := extractOpName name.raw | Macro.throwUnsupported
+    let ovflags ← f.mapM fun f => f.mapM fun
```

Comment:
I don't know what `f` is supposed to be, but I would appreciate clearer variable names, please :)